junit-vintage-engine/src/test/java/org/junit/vintage/engine/DynamicTestsHelperRun.java

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright 2015-2017 the original author or authors.
+ *
+ * All rights reserved. This program and the accompanying materials are
+ * made available under the terms of the Eclipse Public License v1.0 which
+ * accompanies this distribution and is available at
+ *
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+
+package org.junit.vintage.engine;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import org.junit.Assert;
```

Comment:
We do not use `org.junit.Assert`. 

Please use static imports for methods in `org.junit.jupiter.api.Assertions`.